search/includes/classes/class-search.php

```diff
@@ -88,6 +88,7 @@ class Search {
 	private const UPPER_BOUND_QUERY_DB_FALLBACK_VALUE = 10;
 
 	public $healthcheck;
+	public $settingshealthcheck;
```

Comment:
Maybe to stick with the other variables, we could name this `$settings_healthcheck`? Same for the other occurrences on this file.